esu.go

```diff
@@ -11,23 +12,35 @@ type EsConnection struct {
 	Scheme string
 	Host   string
 	Port   string
-	URL    *url.URL
+	URLs   []*url.URL
 	Client *elastic.Client
 }
 
 // New Creates a  ES connection object
 func New(scheme, host, port string) *EsConnection {
 	connection := EsConnection{Scheme: scheme, Host: host, Port: port}
-	connection.URL = getConnectionURL(scheme, host, port)
-	connection.Client = connectToES(connection.URL.String())
+	connection.URLs = append(connection.URLs, getConnectionURL(scheme, host, port))
+	connection.Client = connectToES(connection.URLs)
 
 	return &connection
 }
 
 // NewByUrl Creates a  ES connection object based on elastic url
 func NewByUrl(url string) *EsConnection {
-	connection := EsConnection{}
-	connection.Client = connectToES(url)
+	return NewByUrls(url)
+}
 
+/* NewByUrls creates an ES connection based on a list of comma separated
+urls. */
+func NewByUrls(urls string) *EsConnection {
+	connection := EsConnection{}
+	for _, u := range strings.Split(urls, ",") {
+		parsedUrl, err := url.Parse(u)
+		if err != nil {
+			panic(err)
```

Comment:
Hvorfor ikke retunere er error isteden for panic ? Tror panic i biblioteker er litt utenfor anbefalingen